### PR TITLE
Various mss

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Minor changes.

### CODE REVIEW

**Review by thea-m — APPROVED** [review]

File: ParisBNF/abb/BNFabb39.xml [hunk 1]
```diff
@@ -402,9 +402,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                   <note>According to
                                     <bibl><ptr target="bm:Chaine1912Catalogue"/>
                                     <citedRange unit="page">28</citedRange></bibl> the text is an exceprt from Josippon,
-                                    indeed it appears to correspond to the story of Metitan, son of Judah after the edition <bibl><ptr target="bm:Murad1938ZenaAyhud"/>
-                                    <citedRange unit="page">63</citedRange></bibl>,
-                                    but both are similar only in the beginning. The continuation seems to be only a summary based on <ref type="work" corresp="LIT2598Yosipp">Josipppon</ref>
+                                    it appears to correspond to the story of Metitan, son of Judah after the edition <bibl><ptr target="bm:Murad1938ZenaAyhud"/>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb39.xml:405]: Probably you should add: "At first sight, it seems to correspond ..." - just to make your point clearer and to distance yourself from it, if you feel it appropriate.

**Comment by eu-genia** [comment]
note that this branch has been merged, any changes should be carried out in a new branch

File: ParisBNF/abb/BNFabb39.xml [hunk 2]
```diff
@@ -393,7 +393,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                       <msItem xml:id="p1_i3">
                                          <locus from="63rb" to="65vb"/>
                                          <locus from="1va" to="3vb"/>
-                                         <title type="complete" ref="LIT2598Yosipp" cert="medium"/>
+                                        <title type="incomplete" ref="LIT2598Yosipp" cert="medium">An exceprt from Josippon</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb39.xml:396]: thanks!

File: ParisBNF/abb/BNFabb39.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb39.xml:405]: ok I modified a little. Thanks!

**Pull request merged**